releng-updatesite/category.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<site>
+	<feature id="aobuchow.themes.moderndark.feature">
+		<category name="aobuchow.themes.moderndark.update" />
+	</feature>
+	<category-def name="aobuchow.themes.moderndark.update" label="Modern Dark Theme update site">
```

Comment:
Could you change the label to `Modern Dark Theme` so that it looks a bit nicer in the software install wizard? :)
![image](https://user-images.githubusercontent.com/10300119/84101672-7dff3d00-a9dc-11ea-8d9c-447958b31967.png)

Reply:
renamed